Araali-Tools/GoShelly/.env

```diff
@@ -0,0 +1,10 @@
+MESSAGE=SOMEONE CONNECTED TO GOSHELLY SERVER
+CHANNEL_ID= 
+EMAIL_ID=all@araalinetworks.com
+PORT=443
+SSLCERTGENEMAIL_SERVER=araaligoshellyserver@gmail.com
+SSLCERTGENEMAIL_CLIENT=araaligoshellyclient@gmail.com
```

Comment:
Leave empty

Reply:
Already removed from latest code. Will commit to Araali's repo.